device-scripts/README.md

```diff
@@ -0,0 +1,47 @@
+# Diagnostic Tool for a Device
+
+This is a diagnostic tool that can be used to help diagnose problems
+with devices. It must be run as root because it needs to access 
+network, and TLS keys and in general secure directories
+
+## How to Get It
+The script is available in the lmp-tools repo and can be downloaded and setup like:
+```
+wget https://raw.githubusercontent.com/foundriesio/lmp-tools/master/device-scripts/diag.sh
+chmod 755 diag.sh
+sudo -s
+```
+## How To Run It
+This can be placed in any location on a device, typicallly in either `root` or `fio`.
+
+The best way to execute it is
+```
+# ./diag.sh | tee results.txt
+```
+## Results
+The results will have headers that start with 3 astierisks. The current version of the script
+(currently version 1.0) has the following headings:
```

Comment:
Do you mean the result file is giant, and I have to open it and search for "***" to see the next result?
I'm double-checking only because there is no description of the result itself, only how to get to the next test (headers), and I'm not sure if I understood it as intended